rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2640,3 +2932,5 @@ static void ppu_initialize2(jit_compiler& jit, const ppu_module& module_part, co
 	jit.add(std::move(_module), cache_path);
 #endif // LLVM_AVAILABLE
 }
+
```

Comment:
Undef at the end of cpp file has no effect

Reply:
Remanant of old code, removed.